xlsx-templater.cabal

```diff
@@ -36,18 +36,19 @@ Executable           test
 
   main-is:           Test.hs
 
-  
+  Other-modules:     Codec.Xlsx.Templater
```

Comment:
why not to do it correctly and and dependency from `xlsx-templater` library?
Actually I think it makes sense to make building this executable optional (i.e. enable only when some flag is set on)